sympy/ntheory/factor_.py

```diff
@@ -2201,3 +2201,29 @@ def is_deficient(n):
     if is_perfect(n):
         return False
     return bool(abundance(n) < 0)
+
+
+def is_amicable(n, N):
+    """Returns True if the pair ``(n, N)`` is a Amicable Number, else False.
+
+    Amicable numbers are two different numbers so related that the sum of the proper divisors of each is equal to the other number.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import divisor_sigma
+    >>> is_amicable(3259, 4576)
+    False
+    >>> is_amicable(5020, 5564)
+    True
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Amicable_numbers
+
+    """
+    if((divisor_sigma(n, 1)) - n == N and (divisor_sigma(N, 1)) - N == n ):
```

Comment:
How about this?

```python
 def is_amicable(m, n): 
     a, b = map(lambda i: divisor_sigma(i), (m, n)) 
     return a == b == (m + n) 

```

Reply:
@Yathartha22 but in this code I think  `divisor_sigma` return the sum of all the divisiors including number itself for examble
```
>>> divisor_sigma(220)
504
```
which is not needed for `is_amicable`